Source/References/GameStartr-0.2.0.ts

```diff
@@ -1327,6 +1327,14 @@ declare module GameStartr {
         onGamePause(GameStarter: IGameStartr): void;
 
         /**
+         * Triggered Function for when the game is closed. The game is saved and the
```

Comment:
> The game is saved

No need to include that sentence. This logic is just for GameStartr, not for classes that implement it.